doc/development/guide/installation.rst

```diff
@@ -11,13 +11,18 @@ PennyLane requires the following libraries be installed:
 The following Python packages are hard dependencies, and will automatically
 be installed alongside PennyLane:
 
-* `numpy <http://numpy.org/>`_ >= 1.13.3
-* `scipy <http://scipy.org/>`_ >= 1.0.0
-* `NetworkX <https://networkx.github.io/>`_ >= 1.0.0
+* `numpy <http://numpy.org/>`_ < 2.1
+* `scipy <http://scipy.org/>`_
+* `NetworkX <https://networkx.github.io/>`_
+* `rustworkx <https://github.com/Qiskit/rustworkx>`_ >= 0.14.0
 * `autograd <https://github.com/HIPS/autograd>`_
 * `toml <https://github.com/uiri/toml>`_
 * `appdirs <https://github.com/ActiveState/appdirs>`_
 * `autoray <https://github.com/jcmgray/autoray>`__ >= 0.6.11
+* `cachetools <https://github.com/tkem/cachetools>`_
+* `pennylane-lightning <https://github.com/PennyLaneAI/pennylane-lightning>`_ >= 0.38
+* `requests <https://github.com/psf/requests>`_
+* `typing_extensions <https://github.com/python/typing_extensions>`_
 * `packaging <https://github.com/pypa/packaging>`_
 
 The following Python packages are optional:
```

Comment:
Should we also update the list of optional packages?

For example, matplotlib, quimb, etc ?

Reply:
Is there a source we should draw the list from? I guess I didn't really want to touch that section because it seems very difficult to maintain, but I don't want to remove the mention of OpenFermion if someone added it previosly.